### ISSUE TITLE
after Update of JS-controller 1.5.1 "error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc..."

### ISSUE THREAD

**Issue by MarkSau** [ioBroker/ioBroker.hm-rpc#98]
Hi,
i'm not sure if this error is related to the RPC adapter..but it come from it.
After i've updated the JS-controller to 1.5.1 i got a lot of errors relleated to CuxD.

e.g 

2018-09-14 08:50:07.062 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.LEVEL!
--
2018-09-14 08:50:07.070 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.OLD_LEVEL!
2018-09-14 08:50:07.071 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.STOP!
2018-09-14 08:50:07.072 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.PRESS_SHORT!
2018-09-14 08:50:07.072 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.PRESS_LONG!
2018-09-14 08:50:07.081 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.MOTION!
2018-09-14 08:50:07.085 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.CONTROL!
2018-09-14 08:50:07.085 - error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc.2.CUX4000002.1.SET_STATE!

node: v8.12.0
npm: 6.4.1
js-controller: 1.5.1
rpc: 1.7.6

**Comment by GermanBluefox** [comment]
and If you roll js.controller back to 1.4.2 can you still see the errors?
```
cd /opt/iobroker
iobroker stop
npm i iobroker.js-controller@1.4.2 --production
iobroker start
```

пт, 14 сент. 2018 г. в 9:14, MarkSau <notifications@github.com>:

**Comment by MarkSau** [comment]
I’m back to js-controller 1.5.0 (recover from backup) and I see no errors

 

Von: Bluefox <notifications@github.com> 
Gesendet: Freitag, 14. September 2018 12:31
An: ioBroker/ioBroker.hm-rpc <ioBroker.hm-rpc@noreply.github.com>
Cc: MarkSau <info@mecksauer.de>; Author <author@noreply.github.com>
Betreff: Re: [ioBroker/ioBroker.hm-rpc] after Update of JS-controller 1.5.1 "error: hm-rpc.2 binrpc -> setValue: no dpType for hm-rpc..." (#98)

 

and If you roll js.controller back to 1.4.2 can you still see the errors?
```
cd /opt/iobroker
iobroker stop
npm i iobroker.js-controller@1.4.2 --production
iobroker start
```

**Comment by Apollon77** [comment]
Please also make sure to use the most current versions of hm-rega and hm-rpc!! They are needed when using js.controller 1.5.1!

**Comment by MarkSau** [comment]
I think i had the newest (from latest repository) version installed

 

HomeMatic ReGaHSS:   1.7.1

HomeMatic RPC:              1.7.6

**Comment by MarkSau** [comment]
seams that issue is solved by update of js-controller 1.5.3..now no errors in log found...
thx for your help...
